api/incidents/views.py

```diff
@@ -0,0 +1,25 @@
+from django.shortcuts import render
+from rest_framework import generics, permissions, status, views
+from rest_framework.response import Response
+from .serializers import IncidentSerializer, TABLE
+from django.contrib.auth.models import User
+from .models import Incident
+from rest_framework.permissions import (
+    IsAuthenticatedOrReadOnly, IsAuthenticated, IsAdminUser, AllowAny
+)
+
+# Create your views here.
+
+class CreateRedflagAPIView(generics.CreateAPIView):
+    """
+    post:
+    Create Redflag.
+    """
+
+    serializer_class = IncidentSerializer
+    permission_classes = [AllowAny]
```

Comment:
change thi to:
`permission_classes = [permissions.IsAuthenticated])`
so that only authenticated users can create records

Reply:
I was working on this before the authentication was merged. I have made the change in the new PR.